bundles/binding/org.openhab.binding.zwave/database/popp/05438.xml

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Product>
+	<Model>05438</Model>
+	<Label lang="en">Outdoor Wall Plug Switch 05438</Label>
+	<CommandClasses>
+		<Class>	<id>0x20</id></Class>
+		<Class>	<id>0x25</id></Class>
+		<Class>	<id>0x27</id></Class>
+		<Class>	<id>0x32</id></Class>
+		<Class>	<id>0x59</id></Class>
+		<Class>	<id>0x5A</id></Class>
+		<Class>	<id>0x5E</id></Class>
+		<Class>	<id>0x5E</id></Class>
+		<Class>	<id>0x60</id></Class>
+		<Class>	<id>0x70</id></Class>
+		<Class>	<id>0x72</id></Class>
+		<Class>	<id>0x75</id></Class>
+		<Class>	<id>0x85</id></Class>
+		<Class>	<id>0x86</id></Class>
+		<Class>	<id>0x87</id></Class>
+		<Class>	<id>0x8E</id></Class>
+	</CommandClasses>
+	<Configuration>
+		<Parameter>
+			<Index>1</Index>
+			<Type>list</Type>
+			<Default>1</Default>
+			<Size>1</Size>
+			<Label lang="en">LED indication mode</Label>
+			<Item>
+				<Value>0</Value>
+				<Label lang="en">Disabled</Label>
+			</Item>
+			<Item>
+				<Value>1</Value>
+				<Label lang="en">Show switch state (default)</Label>
+			</Item>
+			<Item>
+				<Value>2</Value>
+				<Label lang="en">Operated by Indicator Command Class</Label>
+			</Item>
+			<Help lang="en">Set LED indication mode.</Help>
+		</Parameter>
+		<Parameter>
+			<Index>2</Index>
+			<Type>int</Type>
+			<Default>0</Default>
+			<Size>1</Size>
+			<Label lang="en">Automatically switch off after x seconds.</Label>
+			<Help lang="en">If not zero, automatically switch device off after a user defined time.
```

Comment:
This will all be merged into a single line. If you want to maintain the formatting, then you need to escape it with CData tags and use the BR tag to terminate lines.